src/magnetVacuumFF.cpp

```diff
@@ -56,42 +56,42 @@ static Ref_t create_detector(Detector& det, xml_h e, SensitiveDetector /* sens *
   double y_b0pf = 0.0*dd4hep::cm;
   double z_b0pf = 575.314*dd4hep::cm;
 
-  double radius_b0apf = 4.3*dd4hep::cm;
+  double radius_b0apf = 4.2*dd4hep::cm;  //cm -- reduce by 1mm
```

Comment:
Reduce compared to what? Or is this note to self?

Reply:
It's a note to self - the original numbers came from some code I use to calculate the numbers, but they are truly exact, and can create some overlaps, so I hand-tuned them (for now) to remove the overlaps. I will improve this setup on the next iteration of the volume definition.